release 0.6.1 notes update (#706)
napari/docs: docs/release/release_0_6_1.md

## Highlights

### The HiLo👋 Colormap!

Introducing the HiLo colormap to napari! 🎨

> This much-loved colormap (LUT) visualizes the maximum contrast limit as red 🔴 and the minimum contrast limit as blue 🔵.

Reviewer: ```suggestion
This much-loved colormap (LUT) is like grayscale, except it displays values above the maximum contrast limit as red 🔴 and values below the minimum contrast limit as blue 🔵.
```

In the scientific imaging world, the HiLo colormap is often used to assess overexposed (saturated) ☀️ and underexposed (dark) 🌑 regions in images.
Enjoy this animation of the HiLo colormap in action! 👇
![HiLo colormap animation](https://github.com/user-attachments/assets/b77e98b4-3f9c-437a-b169-2444544ee454)

> The HiLo colormap is now available as a result of the dependency bump to VisPy 0.15.0 [#7846](https://github.com/napari/napari/pull/7846), and will soon unlock some great new features in the coming napari releases.

Reviewer: ```suggestion
The HiLo colormap is now available as a result of the dependency bump to VisPy 0.15.0 [#7846](https://github.com/napari/napari/pull/7846), which will unlock even more great new features in the coming napari releases.
```
